safe_transaction_service/contracts/tx_decoder.py

```diff
@@ -552,6 +552,34 @@ class DbTxDecoder(TxDecoder):
 
     cache_abis_by_address = TTLCache(maxsize=2048, ttl=60 * 5)  # 5 minutes of caching
 
+    @classmethod
+    def get_most_relevant_contract_abi(cls, address: ChecksumAddress):
+        """
+        Retrieves the most relevant ABI for the contract at the given address.
+
+        :param address: Contract address
+        :return: List of ABI data if found, `None` otherwise.
+        """
+        return (
+            ContractAbi.objects.filter(contracts__address=address)
```

Comment:
In the case that this is required, this method should be a part of `ContractAbi` model `manager` and not of the decoder. But this is on me, as I implemented it that way before and you just moved the code

Reply:
Noted, for the moment I keep it here as it has no logic that requires it to be in the model manager.